### PR TITLE
Sequencer perfomance

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: src/hooks/useStore.js [hunk 1]
```diff
@@ -1,29 +1,132 @@
...
   },
-  
+
+  ///////     init DrumPadPlayers, set PadSettings and Volume    /////////
+  initDrumPadPlayers: () => {
+    const allPads = useStore.getState().allPads;
+    const drumPadPlayersVolume = useStore.getState().drumPadPlayersVolume;
+    const drumPadPlayers = new Tone.Players(
+      {
+        Player0: allPads[0].sample,
```
Comment (Quaquaro) [inline src/hooks/useStore.js:60]: Maybe you vould do something like that.`allPads.map((pad, index)=>'Player${index}:pad.sample)`

**Review by Quaquaro — APPROVED** [review]
Really nice job.

File: src/hooks/useStore.js (hunk 1 above)
Comment (FaGru) [reply src/hooks/useStore.js:60]: Good idea. I will try that

File: src/components/DrumLoopPlayer.js [hunk 2]
```diff
@@ -47,9 +43,9 @@ export default function DrumLoopPlayer({
     setIsPlaying(!isPlaying);
   }
   function handleChange(event) {
-      const currentLoop = event.target.value;
-      getDrumLoop(currentLoop);
-      isPlaying ? setIsPlaying(false) : setIsPlaying(false);
+    const currentLoop = event.target.value;
+    getDrumLoop(currentLoop);
+    isPlaying ? setIsPlaying(false) : setIsPlaying(false);
```
[the remark this answers is not in the document]
Comment (FaGru) [reply src/components/DrumLoopPlayer.js:48]: I do not know. I will check it

**Pull request merged**